services/dask-sidecar/src/simcore_service_dask_sidecar/tasks.py

```diff
@@ -86,32 +91,50 @@ async def _run_computational_sidecar_async(
     log_file_url: AnyUrl,
     command: List[str],
 ) -> TaskOutputData:
+
+    task_publishers = TaskPublisher()
+
     log.debug(
         "run_computational_sidecar %s",
         f"{docker_auth=}, {service_key=}, {service_version=}, {input_data=}, {output_data_keys=}, {command=}",
     )
     current_task = asyncio.current_task()
     assert current_task  # nosec
-    async with monitor_task_abortion(
-        task_name=current_task.get_name(),
-    ):
-        _retry = 0
-        _max_retries = 1
-        sidecar_bootmode = get_current_task_boot_mode()
-        task_max_resources = get_current_task_resources()
-        async with ComputationalSidecar(
-            service_key=service_key,
-            service_version=service_version,
-            input_data=input_data,
-            output_data_keys=output_data_keys,
-            log_file_url=log_file_url,
-            docker_auth=docker_auth,
-            boot_mode=sidecar_bootmode,
-            task_max_resources=task_max_resources,
-        ) as sidecar:
-            output_data = await sidecar.run(command=command)
-        log.debug("completed run of sidecar with result %s", f"{output_data=}")
-        return output_data
+    try:
+        async with monitor_task_abortion(
+            task_name=current_task.get_name(),
+        ):
+            _retry = 0
+            _max_retries = 1
```

Comment:
where might these 2 values be used?

Reply:
this is for now a reminder for the dask-sidecar. since currently there is no retry. so not used.